style.css

```diff
@@ -0,0 +1,455 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Kanit', sans-serif;
+}
```

Comment:
- [ ] You should use the design's font family. Please fix it. 
![image](https://user-images.githubusercontent.com/38635158/176971300-bc2b9ce2-53c5-4cbc-9b83-9d13e7e329e7.png)
[Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.)